lib/taskana-core/src/main/java/pro/taskana/configuration/TaskanaEngineConfiguration.java

```diff
@@ -56,14 +56,15 @@
     private static final String TASKANA_CLASSIFICATION_TYPES_PROPERTY = "taskana.classification.types";
     private static final String TASKANA_CLASSIFICATION_CATEGORIES_PROPERTY = "taskana.classification.categories";
     protected static final String TASKANA_SCHEMA_VERSION = "1.0.2"; // must match the VERSION value in table
-                                                                    // TASKANA.TASKANA_SCHEMA_VERSION
+    // TASKANA_SCHEMA_VERSION
 
     // Taskana properties file
     protected String propertiesFileName = TASKANA_PROPERTIES;
 
     // Taskana datasource configuration
     protected DataSource dataSource;
     protected DbSchemaCreator dbSchemaCreator;
+    protected String defaultSchemaName = "TASKANA";
```

Comment:
We should distinguish more precisely between default value and used value. As I understand this is the actually used value. Then we should name it just "schemaName" or "dbSchemaName". 
We can have a constant defining the default value, which is used if no property is set. But that should a different variable.